src/mocks/fixtures.ts

```diff
@@ -197,3 +198,27 @@ export const submissions: SubmissionMetadata[] = [
     source_client: "field_notes",
   },
 ];
+
+export class FakeErrorTestClient extends FetchClient {
```

Comment:
It's not immediately apparent to me what this class is. I think I'd find a 1-2 line summary of the class helpful (e.g. in a docstring above the class signature).

Reply:
Done